acls.go

```diff
@@ -108,7 +108,9 @@ func (h *Headscale) generateACLPolicySrcIP(u string) ([]string, error) {
 	return h.expandAlias(u)
 }
 
-func (h *Headscale) generateACLPolicyDestPorts(d string) ([]tailcfg.NetPortRange, error) {
+func (h *Headscale) generateACLPolicyDestPorts(
+	d string,
+) ([]tailcfg.NetPortRange, error) {
 	tokens := strings.Split(d, ":")
 	if len(tokens) < 2 || len(tokens) > 3 {
 		return nil, errorInvalidPortFormat
```

Comment:
Is this the standard gofmt config?

Reply:
I mean in general, not just this line...

Reply:
No, its gofumpt,which I think is standard in vscode, and then it is run through golines to break long lines.

Reply:
Added note in readme.